Dockerfile

```diff
@@ -39,6 +39,8 @@ RUN \
   && timedatectl set-timezone Europe/London || true \
   && pip3 install -U setuptools pip wheel
 
+RUN apk add --no-cache gcc libc-dev linux-headers
```

Comment:
Not entirely sure but it may be possible to add these packages to the `apt-get install` above.

Reply:
Thanks @jrmhaig. I tried adding them to the `apt-get install` originally but it didn't work. Your comment made me revisit it. 

I've now replaced `apt-get` with `apk` and as a result removed a number of `ubuntu` packages that aren't necessary/supported on `alpine`, or replaced them with `alpine` equivalents.